libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/UserFeedback.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.core.telemetry.events
+
+import com.mapbox.geojson.Point
+import com.mapbox.navigation.core.MapboxNavigation
+
+/**
+ * Class for user feedbacks, contains properties that were passed to
+ * [MapboxNavigation.postUserFeedback], feedback id and location at that moment.
+ *
+ * @property feedbackId this feedback's unique identifier
+ * @property feedbackType feedback type, one of [FeedbackEvent.Type] or a custom one
+ * @property source one of [FeedbackEvent.Source]
+ * @property description description message
+ * @property screenshot encoded screenshot
+ * @property feedbackSubType array of [FeedbackEvent.SubType] and/or custom feedback subtypes
+ * @property location user location when the feedback event was posted
+ */
+class UserFeedback internal constructor(
```

Comment:
That's public API, right? How do you keep track of classes that are API and that aren't? By using internal modifier? Or is there anything else? Packages maybe?